### ISSUE TITLE
Check spelling of input text

### ISSUE THREAD

**Issue by Amr-Ibra** [dialect-app/dialect#413]
* dialect 2.5.0

Input text is not checked for spelling mistakes. Could you please support that?

https://gitlab.gnome.org/GNOME/libspelling

**Comment by rafaelmardojai** [comment]
I've considered this before but we would want check spelling for the selected language, and `libspelling` will use the dictionaries installed locally, that usually means English and you local language.

But anyway, I guess It would still be nice to have check spelling for English and the user local language.

@mufeedali what do you think?

**Comment by mufeedali** [comment]
> But anyway, I guess It would still be nice to have check spelling for English and the user local language.

Yes, I think that would be the best since it's not possible to support every language anyway.

**Comment by Amr-Ibra** [comment]
I don't think you support specific languages explicitly. `ibspelling` uses `Enchant`, which uses whatever `Hunspell` or `Aspell` dictionaries that the user has installed.

**Comment by rafaelmardojai** [comment]
@Amr-Ibra yeah, we're just talking about how our ideal spell checking implementation would work, where we would like spell checking for every language that can be selected in the input without having the user to install additional packages.

**Comment by Amr-Ibra** [comment]
On Debian, `Hunspell` dictionaries are system-wide, not individually installed for every app. If the user has no `Hunspell` dictionaries installed, `libspelling` will not be able to find any spelling mistakes.

**Comment by rafaelmardojai** [comment]
@Amr-Ibra yes, as I said we're talking about something we would like but is not possible using libspelling and won't happen any time soon. So just ignore that 😅, we WILL implement spell checking using libspelling with the limitations it has.
